### ISSUE TITLE
Display contracts' source code in explorer

### ISSUE THREAD

**Issue by mvaivre** [alephium/alephium-frontend#355]
To allow for more transparency, it would be great to display the contracts' source code in the explorer, similar to what etherscan is doing: https://etherscan.io/token/0x590F820444fA3638e022776752c5eEF34E2F89A6#code

**Comment by mvaivre** [comment]
@tdroxler do you know if the current API allows for this?

**Comment by tdroxler** [comment]
nope, we only store byte code, we don't have access to source code if  developers don't share it
